translations/ui-export-manager/en_GB.json

```diff
@@ -38,5 +38,7 @@
     "exportJob.details.action.resend.success": "Your file upload has started successfully.",
     "permission.jobs.downloadAndResend": "Export manager: Download and re-send files",
     "exportJob.type.BULK_EDIT": "Bulk edit",
+    "exportJob.type.BULK_EDIT_IDENTIFIERS": "Bulk edit identifiers",
```

Comment:
you don't need to make changes in any translation files except en.json

Reply:
Okay, I will remove that. I have done it because we had like agreement in our team, that when we adding new translation, we was adding it in en_US, en_GB and en

Reply:
translation script upgrades everything except en.json, there are case when it ignores, in this case it should be pointed manually

Reply:
Okay, thanks for that point.